### PR TITLE
Add install requirements.txt

### PR DESCRIPTION
**Pull request by All4Gis**
why not install requeriments.txt for example pypubsub==3.3,that is necessary for use it,and only install requeriments-dev?

### CODE REVIEW

File: pavement.py [hunk 1]
```diff
@@ -181,6 +181,16 @@ def install_devtools():
 
     pip.main(['install', '-r', 'requirements-dev.txt'])
 
+@task
+def install_tools():
+    """Install development tools"""
```
Comment (All4Gis) [inline pavement.py]: """Install tools"""

**Comment by luipir** [comment]
I don't understand this the PR... is it not enough do "paver setup" to install requirements.txt? what is the utility to add a new task? what is the use case?

**Comment by All4Gis** [comment]
It is a strange case but in some machines it does not install the project dependencies, 
but I have been unable to reproduce it in my machine. However, to give the user another way to install the necessary and the correct 
version of the resources without the need of much knowledge, I add this method just in case they do not work.

In the case that paver does not install the dependencies, you can install it using 

> pip install -r requeriments.txt

 and work,then this method only execute this.

As I said, this is a strange case and maybe I shouldn't report it as a PR, but as a bug, but as I mentioned, I haven't been able to reproduce it.

**Comment by gioman** [comment]
> It is a strange case but in some machines it does not install the project dependencies,

it cannot be related at all to the code: at home I have a land connection that simply does not allows me to install (all) dependencies with paver, for some reason. If I switch to another connection it just works.

**Comment by luipir** [comment]
befeore to add a feature, can we have an opened issue to explore? @gioman says that it can depends on connection type, @All4Gis says that he's not able to reproduce. first step is to try to have a reproducible  issue procedure.

**Comment by All4Gis** [comment]
I can open an issue if it's better. The trick is to get that method to try to reproduce it.

**Comment by All4Gis** [comment]
Close this PR or merge?Any decision?

**Pull request closed without merging**